configure.ac

```diff
@@ -0,0 +1,39 @@
+# version parameter is unused
+AC_INIT([MicroOVN], [1.0])
+
```

Comment:
We should probably declare a minimal version of `autoconf` with `AC_PREREQ`.

Reply:
I'm genuinely interested in best practice for autoconf here. It seems to me bit counterproductive to try to keep minimum compatible version, because it's likely to get out of sync if future contributors don't pay close attention. Overall the autoconf runs as one of the first things when bootstrapping project, and it tells you if you are missing dependencies, wouldn't it be enough if it reported incompatible/unknown macros on its own?

Anyway, I tried to look up macros we are using in the `NEWS` file of autoconf, and for most of them I couldn't even find when they were added (mostly when they were documented). To me it seems that the this file is compatible with every autoconf in past ~20 years.

Reply:
Ok, let's hope we remember to add it whenever we make use of a new feature.